app/js/components/orderEntry/styles.scss

```diff
@@ -171,3 +171,21 @@
     max-height: 500px;
   }
 }
+
+.draft-action-btns {
+  display: flex;
+  span {
+    width:40%;
+    a {
+      i.scale {
+        text-decoration: none;
+         color: black;
+      }
+    }
+    :hover{
+      transform: scale(1.5);
+      -webkit-transform: scale(1.5);
+      -moz-transform: scale(1.5);
+    }
+  }
+}
```

Comment:
Some concerns on this class name`draft-action-btns`.
The styles inside the class is actually affecting some elements on the orders home page, I understand that the same styles may also be needed for the draft, but you should consider using a more generic name that would reflect the fact that the class is not for the draft action buttons alone.

Reply:
@IAMOTZ I don't see how it would affect anything else anywhere since the stylesheet itself is scoped to `OrderEntryPage.jsx` and yes the styles are for the buttons on the home page ? am I missing something

Reply:
Hey @rhenshaw56 
I was not talking about the effect of the styles, the comment was only about the name of the class holding the styles `draft-action-btns`. Since the styles are both used on the `Draft` and the `Orders` view, I was suggesting you use a more generic name like  `order-action-btn`.
Do you get?

Reply:
done.